gh-pages-stub/public/js/search.js

```diff
@@ -0,0 +1,93 @@
+(function() {
+
+  function isBlank(str) {
+    return (!str || /^\s*$/.test(str));
+  } 
+
+  function displaySearchResults(results, store, searchTerm) {
+    var searchResults = document.getElementById('search-results');
+
+    if (results.length) {
+      var appendString = '';
+
+      for (var i = 0; i < results.length; i++) {
+        var item = store[results[i].ref];
+        if (isBlank(item.title)) {
+            item.title = "No title";
+        }
+        appendString += '<li><a href="' + item.url + '"><h3>' + item.title + '</h3></a>';
```

Comment:
i think better to use reduce for this
appendString = reduce((prev, cur) => prev + '<li><a href="' + item.url + '"><h3>' + item.title + '</h3></a>') - something like this, but a little more complex